recipe/meta.yaml

```diff
@@ -132,27 +158,31 @@ outputs:
 
         Supported Python Versions
         -------------------------
-        Python >= 3.6
+        Python >= 3.8
 
         Deprecated Python Versions
         --------------------------
-        Python == 2.7, Python == 3.5.
+        Python == 2.7, Python <= 3.6.
 
         The last version of this library compatible with Python 2.7
         and 3.5 is google-cloud-bigquery-storage==1.1.0.
-      doc_url: https://googleapis.dev/python/bigquerystorage/latest/index.html
+      doc_url: https://cloud.google.com/python/docs/reference/bigquerystorage/latest/summary_overview
       dev_url: https://github.com/googleapis/python-bigquery-storage
 
 about:
```

Comment:
Do we need the `about` section here if there is one in each output?

Reply:
I left the structure from conda-forge to avoid too many future diff changes